warehouse/templates/pages/help.html

```diff
@@ -518,6 +520,26 @@ <h3 id="username-change">{{ username_change() }}</h3>
         <p>{% trans %}PyPI does not currently support changing a username.{% endtrans %}</p>
         <p>{% trans %}Instead, you can create a new account with the desired username, add the new account as a maintainer of all the projects your old account owns, and then delete the old account, which will have the same effect.{% endtrans %}</p>
 
+        <h3 id="openid-connect">{{ openid_connect() }}</h3>
+        <p>
+          {% trans openid="https://openid.net/connect/", apitoken="#apitoken" %}
+          PyPI users and projects can use <a href="{{ openid }}">OpenID Connect</a> to delegate
```

Comment:
```suggestion
          PyPI users and projects can use <a href="{{ openid }}">OpenID Connect</a> (OIDC) to delegate
```
Also, is `openid` supposed to be a URL here?

Reply:
Yep, I'll fix that.

Reply:
Oh, it is -- it's just templated in because this whole block is in a translated section 🙂

Reply:
Ah, I missed that, sorry!